packages/amplication-server/project.json

```diff
@@ -72,6 +72,7 @@
     "serve": {
       "executor": "@nrwl/node:node",
       "options": {
+        "port": 7000,
```

Comment:
what is that for?

Reply:
debugging port. This in conjunction with the attach debugging server launch config, will let us debug the serve app,